articles/reliability/reliability-fabric.md

```diff
@@ -171,7 +171,7 @@ When a major disaster renders the primary region unrecoverable, Microsoft Fabric
 
 The time it takes for failover to complete can vary, although it typically takes less than one hour. Once failover is complete, here's what you can expect:
 
-* **Fabric portal**: You can access the portal, and read operations such as browsing existing workspaces and items continue to work. All write operations, such as creating or modifying a workspace, are paused.
+* **Fabric portal**: You can access the portal, and read operations such as browsing existing workspaces, task flows in workspaces and items continue to work. All write operations, such as creating or modifying a workspace, are paused.
```

Comment:
Can you change the sentence as follows:
* **Fabric portal**: You can access the portal, and read operations, such as browsing existing workspaces, task flows in workspaces, and items, continue to work. All write operations, such as creating or modifying a workspace, are paused.